api/database/migrations/sql/20190908_remove_dead_tables.sql

```diff
@@ -0,0 +1,22 @@
+/*
+ * Remove tables that are unused
+ */
+DROP TABLE subscription;
```

Comment:
You've removed the seed files. Won't this crash if run on a new db?

I imagine this needs to be:
```sql
DROP TABLE subscription IF EXISTS;
```